src/custom-elements.js

```diff
@@ -635,6 +646,21 @@ let Deferred;
     }
   }
 
+  /**
+   * @param {!Map<string, !CustomElementDefinition>} definitions
+   * @param {!Node|!HTMLElement|null} node
```

Comment:
I do not know much about Closure compiler's type system, but here's my attempt. This line of code causes warnings because Closures definition for `Node` doesn't have a `tagName` nor a `is` property.

Do you need me to add types to everything? Also do you need me to fix all type warnings? If so I would appreciate help on doing so